device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/mqtt/Mqtt.java

```diff
@@ -32,8 +32,9 @@
     private static final int QOS = 1;
     private static final int MAX_SUBSCRIBE_ACK_WAIT_TIME = 15 * 1000;
 
-    // paho mqtt only supports 10 messages in flight at the same time
-    private static final int MAX_IN_FLIGHT_COUNT = 10;
+    // relatively arbitrary, but only because Paho doesn't have any particular recommendations here. Just a high enough
+    // value that users who are building a gateway type solution don't find this value to be a bottleneck.
+    static final int MAX_IN_FLIGHT_COUNT = 65000;
```

Comment:
what is the impact of removing `private`? What is the default?

Reply:
The default is "package private" which means it can now be referenced by other classes in the same package. Since we use this value in a different class in the same package, this modifier change is needed

Reply:
In .NET, we prefer to be explicit rather than assume defaults. Not so in Java?

Reply:
There is no explicit way to declare a field/method "package private"  in Java other than to have no access modifier

```java
private String myPrivateField;
String myPackagePrivateField;
protected String myInheritibleField;
public String myFullyPublicField;
```